pkgs/servers/mail/opensmtpd/default.nix

```
@@ -65,6 +65,7 @@ stdenv.mkDerivation rec {
     license = licenses.isc;
     platforms = platforms.linux;
     maintainers = with maintainers; [ rickynils obadz ekleog ];
+    knownVulnerabilities = [ "CVE-2020-8794 / NixOS PR #80978 has not been backported. Upgrade to NixOS 20.03." ];
```

```suggestion
    knownVulnerabilities = [ "CVE-2020-8794" ];
```

I don’t think knownVulnerabilities is used for giving advice, and it definitely can’t recommend people upgrade to a pre-release NixOS release.